readme.md

```diff
@@ -1,59 +1,53 @@
 # SML Portal
 
-This repo contains the build environment and code to generate and upload the Statistical Methods Library (SML) Portal website.
+This repo contains the build environment and code to generate and upload the Statistical Methods Library (SML) Portal Web Application.
 
-## Build and deploy
+## Prerequisites
 
-### S3 Bucket creation
+Before continuing, install the following tools:
 
-[Terraform](https://www.terraform.io/intro) is used to deploy the S3 Bucket where the website is hosted. The S3 bucket will be suffixed with the currently checked-out gt branch name.
+[**Homebrew**](https://brew.sh/): Follow the instructions on this page
 
-Fetch your AWS credentials and export them to the terminal. Then:
+[**Python 3.10.0**](https://www.python.org/downloads/macos/): To install python on your local system, you can visit this link and install the relevant version (Python 3.10.0)
 
-```shell
-./terraform.sh apply
-```
+You will need to install pipenv to install the python dependencies and pyenv if you want to manage multiple versions of python. Refer to the [confluence page](https://confluence.ons.gov.uk/pages/viewpage.action?pageId=14902311) for guidance on this.
```

Comment:
Didn't we also need to install wget so that the design-system could be fetched?

Reply:
Yes, that's correct. I added this in now =)